clang/include/clang/Basic/TokenKinds.h

```diff
@@ -101,6 +101,13 @@ inline bool isLiteral(TokenKind K) {
          isStringLiteral(K) || K == tok::header_name || K == tok::binary_data;
 }
 
+/// Return true if this is a utf literal kind.
```

Comment:
Minor nit: Use "UTF" (all caps).
```suggestion
/// Return true if this is a UTF literal kind.
```